hw/top_darjeeling/data/xbar_dbg.hjson

```diff
@@ -17,18 +17,31 @@
       xbar:       "true",
       pipeline:   false,
     },
+    { name:      "rv_dm.dbg",
+      type:      "device",
+      clock:     "clk_dbg_i",
+      reset:     "rst_dbg_ni",
+      pipeline:  false,
+    },
     { name:       "mbx_jtag.soc",
       type:       "device",
-      addr_space: "soc_dbg",
```

Comment:
Why is this particular line removed? Is it sufficient to have it only on the host port?

Reply:
Yes, address spaces are properties of hosts only. A host issues an address to the interconnect, and the xbar transforms it from the host's space to the corresponding device's register block space (i.e. to a selection signal and offset).

The same device register block could be addressed from multiple host spaces, if the top-level hjson had multiple entries for the related dictionary and an xbar hjson connected a host from each. At this point, we don't do that, though--That feature may still have some wrinkles for our DV setup, haha.